website/docs/components/side-nav/partials/accessibility/accessibility.md

```diff
@@ -17,7 +17,7 @@ The `Component Name` component is not WCAG-conformant on its own. (Explain how t
 
 This section is for reference only. This component intends to conform to the following WCAG Success Criteria:
 
-<Doc::WcagList @criteriaList={{array "..." }} />
+<Doc::WcagList @criteriaList={{array "1.1.1" "1.3.1" "1.3.2" "1.3.3" "1.3.4" "1.4.1" "1.4.3" "1.4.4" "1.4.10" "1.4.11" "1.4.12" "1.4.13" "2.1.1" "2.1.2" "2.1.4" "2.4.3" "2.4.7" "3.2.1" "3.2.3" "3.2.4" "4.1.2" }} />
 
```

Comment:
@MelSumner I added the relevant WCAG criteria from your list in the CRD.

What conformance rating badge and related info should I include above though? (I assume that "Conditionally conformant" would be the most appropriate?)

Reply:
I think we can call it conformant? the home link seems to be missing aria-label though.

Reply:
Ok thanks! I think I should add an ariaLabel arg and make it required for the HomeLink as I did for the SideNav::IconButton component. I'll see if I can do that as part of this PR.

Reply:
> The home link seems to be missing aria-label, though.

@MelSumner The idea was that, since we're spreading the `...attributes`, would be responsibility of the developers to do it. If instead we want to make it a required element that triggers an error/assertion, than it's OK for me.

> I'll see if I can do that as part of this PR.
@KristinLBradley please open a different PR for this (pointing to `main`). Thanks

Reply:
Since we are covering this for consumers in other components, I think it would be good to do the same here. I don't mind if they want to provide a localized string and we provide the fallback, but something should exist. 👍 

I think it would actually be okay to hardcode the company name, since that is the same in any language. It _might_ throw an error if they have their linter turned on for "no bare strings" (essentially, apps can decide they want to force localization/internationalization). I think it's also maybe the same for our product names. I'm not sure they would ever change per language. 🤔

Reply:
This is designed to take different logos/icons to represent the different products. SO for example if Boundary uses it I would assume they would customize it to display the Boundary icon and would want the aria-label text to say "Boundary".

I could be wrong though but that was my understanding.

Reply:
Yes, @KristinLBradley you're correct. Different products may need different aria labels.